### ISSUE TITLE
Error code preventing alerts sign-up!

### ISSUE THREAD

**Issue by HelenCEBM** [ebmdatalab/openprescribing#862]
Two users have reported being unable to sign up for alerts. 
I just tried and got this error after inputting email address and clicking `sign up`:
![image](https://user-images.githubusercontent.com/24392156/40482603-44893a84-5f4d-11e8-9d0f-f5c307e85af2.png)

**Comment by inglesp** [comment]
Here's the exception:

```
2018-05-24 12:44:17,633 ERROR [django.request:284] base 15135 140544351328000 Internal Server Error: /ccg/11M/
Traceback (most recent call last):
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/newrelic-2.78.0.57/newrelic/hooks/framework_django.py", line 503, in wrapper
    return wrapped(*args, **kwargs)
  File "/webapps/openprescribing/openprescribing/frontend/views/views.py", line 281, in measures_for_one_ccg
    'pct')
  File "/webapps/openprescribing/openprescribing/frontend/views/views.py", line 426, in _handleCreateBookmark
    signup=True)
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/allauth/account/utils.py", line 145, in perform_login
    send_email_confirmation(request, user, signup=signup)
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/allauth/account/utils.py", line 311, in send_email_confirmation
    signup=signup)
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/allauth/account/models.py", line 58, in send_confirmation
    confirmation.send(request, signup=signup)
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/allauth/account/models.py", line 122, in send
    get_adapter(request).send_confirmation_mail(request, self, signup)
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/allauth/account/adapter.py", line 396, in send_confirmation_mail
    ctx)
  File "/webapps/openprescribing/openprescribing/frontend/account/adapter.py", line 20, in send_mail
    msg.send()
  File "/webapps/openprescribing/openprescribing/frontend/models.py", line 733, in send
    self.message.send()
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/newrelic-2.78.0.57/newrelic/api/function_trace.py", line 110, in literal_wrapper
    return wrapped(*args, **kwargs)
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/core/mail/message.py", line 292, in send
    return self.get_connection(fail_silently).send_messages([self])
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/core/mail/message.py", line 250, in get_connection
    self.connection = get_connection(fail_silently=fail_silently)
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/core/mail/__init__.py", line 36, in get_connection
    klass = import_string(backend or settings.EMAIL_BACKEND)
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/utils/module_loading.py", line 27, in import_string
    six.reraise(ImportError, ImportError(msg), sys.exc_info()[2])
  File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/utils/module_loading.py", line 23, in import_string
    return getattr(module, class_name)
ImportError: Module "anymail.backends.mailgun" does not define a "MailgunBackend" attribute/class
```

**Comment by inglesp** [comment]
This is in the settings:

```
289:EMAIL_BACKEND = "anymail.backends.mailgun.MailgunBackend"
```

**Comment by inglesp** [comment]
Looks like this is the old name for the setting, and it should now be: `anymail.backends.mailgun.EmailBackend`.

**Comment by inglesp** [comment]
The version of anymail we use changed in 458e4f5243ba8bf794fcddd663ec0c7bc47aa6ab.

**Comment by inglesp** [comment]
Fixed in #863.
